server/controllers/followersController.js

```diff
@@ -36,7 +36,18 @@ exports.getFollowersUnfollowers = function (req, res) {
 
 }
 
-
+// GET /user/:userId
```

Comment:
I think we should add '/followers' at the tail for a better understanding of what the controller does. I mean the naming seems to get only user data with userId

Reply:
Strongly agree ! We should convert **/user/:userId** into /user/:userId/followers